src/Compilers/Core/Portable/InternalUtilities/CompilerFeatureRequiredAttribute.cs

```diff
@@ -5,7 +5,7 @@
 // Copied from:
 // https://github.com/dotnet/runtime/blob/fdd104ec5e1d0d2aa24a6723995a98d0124f724b/src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/CompilerFeatureRequiredAttribute.cs
 
-#if !NET6_0_OR_GREATER
```

Comment:
Would `#if NETSTANDARD` be better, so that it is not updated every time we bump version?

Reply:
seems this type is not in .net 6 either, only in net 7

Reply:
Here's the runtime PR addign those types
https://github.com/dotnet/runtime/pull/64287